presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -73,11 +75,11 @@
     private static final String NAME_PROPERTY = "access-control.name";
 
     private final TransactionManager transactionManager;
-    private final File configFile;
+    private final List<File> configFiles;
     private final Map<String, SystemAccessControlFactory> systemAccessControlFactories = new ConcurrentHashMap<>();
     private final Map<CatalogName, CatalogAccessControlEntry> connectorAccessControl = new ConcurrentHashMap<>();
 
-    private final AtomicReference<SystemAccessControl> systemAccessControl = new AtomicReference<>(new InitializingSystemAccessControl());
+    private final AtomicReference<List<SystemAccessControl>> systemAccessControls = new AtomicReference<>(ImmutableList.of(new InitializingSystemAccessControl()));
```

Comment:
Instead of introducing a list and managing complexity in this class, I'd rather see `CompositeSystemAccessControl`. More code lines in total, but `AccessControlManager` would be simpler, and it already handles some other responsibilities.

Reply:
I don't think it is needed, but someone could take a look at it.  BTW, with my latest changes adding multiple here is even easier.

Reply:
I added https://github.com/prestosql/presto/pull/2178/commits/204b52b7122e7e1392e14396f1e9fabdf4999225 . Feel free to include or drop it.

Reply:
I feel that `CompositeSystemAccessControl` adds complexity without providing any value (it removes a few simple for-each loops in `AccessControlManager`). It's basically a helper class for `AccessControlManager`, which still has to deal with loading multiple managers.